openedx/openedx-platform
# fix: show answer should show correct answers

## Description

For problem blocks with ids that contain periods (ie. Question7.1.1), the Show Answer button does not display the green check mark that shows the correct answers. This is due to how jQuery finds elements with the id.

## Supporting information

https://2u-internal.atlassian.net/browse/TNL-11466

## Testing instructions

1. Import a course where problem blocks have ids with periods. (Creating a problem block will not create an id with a period.)
2. Run `paver update_assets cms` in the cms shell to update assets.
3. Answer the problem with the wrong answer.
4. Click 'Show Answer'. You should see green check marks on the correct answers.
Without the code change, there will not be green check marks.

Screenshot of Issue:
<img width="861" alt="image" src="https://github.com/openedx/edx-platform/assets/56318341/47ef03a5-9871-4fb3-b402-7abe6ae9f2e7">

Screenshot of Correct Behavior:
<img width="910" alt="image" src="https://github.com/openedx/edx-platform/assets/56318341/4c0eb81b-12aa-413b-899f-b0ed6fa58cd2">

Note:
You can quickly see the problem id with inspect element
<img width="367" alt="image" src="https://github.com/openedx/edx-platform/assets/56318341/6704edda-5bb5-401f-9a4b-1a3af7eaee89">
